.circleci/config.yml

```diff
@@ -177,10 +173,6 @@ workflows:
       - util_job:
           name: Code Linting
           steps:
-            - run:
```

Comment:
Probably it's best to let Eastwood run last (as it's the slowest linter of the bunch), for a faster feedback loop